src/assets/scripts/airport/Arrival/ArrivalBase.js

```diff
@@ -61,150 +187,208 @@ export default class ArrivalBase {
             this.heading = degreesToRadians(options.heading);
         }
 
+        // TODO: is altitude ever not a number?
```

Comment:
In some of the implementations, you could supply a range of altitudes at which to spawn aircraft, for example `[10000, 15000]`, or simply a singular altitude, `13000`. This block is just to keep the format of `this.altitude` consistent.